src/main/webapp/index.html

```diff
@@ -0,0 +1,3 @@
+<head>
+    <meta http-equiv="refresh" content="0; url=readfile.html" />
+</head>
```

Comment:
Can you add the `<html>` element and an empty `<body>` for now with the doctype?